services/cinema_hall.py

```diff
@@ -0,0 +1,20 @@
+from django.db.models import QuerySet
+
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> QuerySet:

```

Comment:
Indicate in the type annotation a `QuerySet` of what instances your return here and in the other places as well